README.md

```diff
@@ -55,9 +51,9 @@ LDK/`rust-lightning` is a generic library which allows you to build a Lightning
 node without needing to worry about getting all of the Lightning state machine,
 routing, and on-chain punishment code (and other chain interactions) exactly
 correct. Note that `rust-lightning` isn't, in itself, a node. There are various
-working/in progress demos which could be used as a node today, but if you "just"
-want a generic Lightning node, you're almost certainly better off with [Core
-Lightning](https://github.com/ElementsProject/lightning) or
+nodes built using LDK such as [Sensei](https://l2.technology/sensei), but if you
+"just" want a generic Lightning node, you're almost certainly better off with
```

Comment:
Maybe using the term "binary" rather than "generic Lightning node" would get the point across better.

Reply:
or "complete daemon binary"  :bike: